### ISSUE TITLE
1.0.7 Bug in twisted.web.client.py - Patch

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7583]
|<img alt="malamute's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| malamute reported|
|-|-|
|Trac ID|trac#289|
|Type|defect|
|Created|2003-09-26 17:30:49Z|

Attachments:

* [twisted.web.client.py.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/6b0/6b0f4d999089662690c5233e0ddea57d297a9a0a/556aabb3abdd9b6a87adc955d19b6335fd5a4cd2.patch) (864 bytes) - added by malamute on 2003-09-26 17:30:49Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__289 289
type__defect defect
reporter__malamute malamute
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1064597449000000 1064597449000000
changetime__1467625570215875 1467625570215875
version__ 
owner__ 
cc__malamute
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="malamute's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|malamute commented|
|-|-|

```
#!html
<pre>
Found a little bug in twisted.web.client.py and have a 
patch for it.

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub commented|
|-|-|

In changeset 4e139fe239ff0fcc288a87460d772feca2b1ae8c
```
#!CommitTicketReference repository="" revision="4e139fe239ff0fcc288a87460d772feca2b1ae8c"
Merge pull request #289 from twisted/8557-rodrigc-trial

Author: rodrigc
Reviewer: Lukasa
Fixes: #8557

Replace apply() with a function call. apply() is gone in Python 3
```
